src/Sulu/Bundle/MediaBundle/Tests/Functional/Media/Storage/S3StorageTest.php

```diff
@@ -19,14 +19,15 @@
 
 class S3StorageTest extends SuluTestCase
 {
-    /**
-     * @var string
-     */
-    protected static $class = S3Kernel::class;
+    protected function setUp(): void
+    {
+        static::$class = S3Kernel::class;
+    }
 
-    protected static function getKernelClass(): string
+    protected function tearDown(): void
     {
-        return S3Kernel::class;
+        static::$class = null;
```

Comment:
@alexander-schranz this is already done in the [KernelTestCase](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php#L37)

Reply:
Class is missed on lowest version: https://github.com/symfony/symfony/blob/v5.4.0/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php#L49